packages/tutorial/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "tutorial",
+  "version": "1.0.0",
+  "volta": {
+    "extends": "../../package.json"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "^2.0.1",
+    "@types/html-to-text": "^9.0.1",
+    "@types/lodash": "^4.14.195",
+    "@types/node": "^20.3.1",
+    "@types/prompt-sync": "^4.2.0",
+    "@types/uuid": "^9.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.59.6",
+    "@typescript-eslint/parser": "^5.59.6",
+    "eslint": "^8.40.0",
+    "eslint-config-nth": "^2.0.1",
+    "typescript": "^5.1.3"
+  },
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "part1": "yarn build && node dist/part1.js",
+    "part2": "yarn build && node dist/part2.js",
+    "part3": "yarn build && node dist/part3.js",
+    "part4": "yarn build && node dist/part4.js",
+    "view-logs": "cat ai-jsx.log | pino-pretty",
+    "lint": "eslint . --max-warnings 0",
+    "lint:fix": "eslint . --fix",
+    "typecheck": "tsc -p tsconfig.json",
+    "build": "yarn run typecheck"
+  },
+  "dependencies": {
+    "ai-jsx": "^0.2.0-4",
+    "axios": "^1.4.0",
+    "html-to-text": "^9.0.5",
```

Comment:
suggest using `turndown` so we can get Markdown output (this is what I was going to do in the docsqa sample next)